### PR TITLE
Update scenario test suite for IS 7.0.0

### PR DESCRIPTION
**Pull request by Lakshan-Banneheke**
This PR is to update the scenario test suite to be in line with the IS 7.0.0. The scenario test suite was executed on TestGrid and all tests passed.

### CODE REVIEW

**Review by VivekVinushanth — DISMISSED** [review]

File: product-scenarios/4-single-sign-on/4.1-sso-for-web-app/4.1.4-sso-with-oidc/4.1.4.3-basic-profile/src/test/java/org/wso2/identity/scenarios/sso/test/oidc/OIDCSSOTestCase.java [hunk 1]
```diff
@@ -233,52 +207,4 @@ public void introspectAccessTokenForApp1() throws Exception {
 
         httpCommonClient.consume(response);
     }
-
```
[the remark this answers is not in the document]
Comment (Lakshan-Banneheke) [reply product-scenarios/4-single-sign-on/4.1-sso-for-web-app/4.1.4-sso-with-oidc/4.1.4.3-basic-profile/src/test/java/org/wso2/identity/scenarios/sso/test/oidc/OIDCSSOTestCase.java:236]: That has been added here: https://github.com/Lakshan-Banneheke/product-is/blob/99f9da234b11de58ce866ac974bc4b7671da0897/product-scenarios/4-single-sign-on/4.1-sso-for-web-app/4.1.4-sso-with-oidc/4.1.4.3-basic-profile/src/test/java/org/wso2/identity/scenarios/sso/test/oidc/OIDCSSOTestCase.java#L101

The test is a duplicate with only the json file changed, hence did it this way.

File: product-scenarios/scenarios-commons/src/main/java/org/wso2/identity/scenarios/commons/OAuth2CommonClient.java [hunk 2]
```diff
@@ -60,10 +60,8 @@
 import static org.testng.Assert.assertNotNull;
 import static org.testng.Assert.assertTrue;
 import static org.wso2.carbon.utils.multitenancy.MultitenantConstants.SUPER_TENANT_DOMAIN_NAME;
-import static org.wso2.identity.scenarios.commons.util.Constants.CONTENT_TYPE_APPLICATION_FORM;
-import static org.wso2.identity.scenarios.commons.util.Constants.CONTENT_TYPE_APPLICATION_JSON;
-import static org.wso2.identity.scenarios.commons.util.Constants.HTTP_RESPONSE_HEADER_LOCATION;
-import static org.wso2.identity.scenarios.commons.util.Constants.IS_HTTPS_URL;
+import static org.wso2.identity.scenarios.commons.ScenarioTestBase.getDeploymentProperty;
+import static org.wso2.identity.scenarios.commons.util.Constants.*;
```
Comment (lashinijay) [inline product-scenarios/scenarios-commons/src/main/java/org/wso2/identity/scenarios/commons/OAuth2CommonClient.java:64]: Shall we avoid * imports?

File: product-scenarios/scenarios-commons/src/main/java/org/wso2/identity/scenarios/commons/SSOCommonClient.java [hunk 3]
```diff
@@ -88,6 +89,10 @@ public String getSessionDataKey(HttpResponse response) throws Exception {
         return null;
     }
 
+    public String getSessionDataKeyFromURLParam(String url) throws URISyntaxException {
+        return DataExtractUtil.getParamFromURIString(url, "sessionDataKey");
```
Comment (lashinijay) [inline product-scenarios/scenarios-commons/src/main/java/org/wso2/identity/scenarios/commons/SSOCommonClient.java:93]:
```suggestion

        return DataExtractUtil.getParamFromURIString(url, "sessionDataKey");
```

File: product-scenarios/scenarios-commons/src/main/java/org/wso2/identity/scenarios/commons/OAuth2CommonClient.java (hunk 2 above)
Comment (Lakshan-Banneheke) [reply product-scenarios/scenarios-commons/src/main/java/org/wso2/identity/scenarios/commons/OAuth2CommonClient.java]: PR updated

File: product-scenarios/scenarios-commons/src/main/java/org/wso2/identity/scenarios/commons/SSOCommonClient.java (hunk 3 above)
Comment (Lakshan-Banneheke) [reply product-scenarios/scenarios-commons/src/main/java/org/wso2/identity/scenarios/commons/SSOCommonClient.java:94]: PR updated

**Pull request merged**